optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/comparable/TestdataComparableEntity.java

```diff
@@ -0,0 +1,54 @@
+package org.optaplanner.core.impl.testdata.domain.comparable;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.domain.entity.descriptor.EntityDescriptor;
+import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
+import org.optaplanner.core.impl.domain.variable.descriptor.GenuineVariableDescriptor;
+import org.optaplanner.core.impl.testdata.domain.TestdataObject;
+
+@PlanningEntity(difficultyComparatorClass = TestdataCodeComparator.class)
+public class TestdataComparableEntity extends TestdataObject {
```

Comment:
It's named ComparableEntity but doesn't implement Comparable?

Reply:
Yes, it does sound a bit confusing. I've found it fitting for the use-case at that time, but it does break the Comparable naming convention. I've changed the entity to TestdataEntityWithDifficultyComparator and the solution to TestdataSolutionWithDifficultyComparatorEntity. I've wanted to fit in "Class" and make them into TestdataEntityWithDifficultyComparatorClass and TestdataSolutionWithDifficultyComparatorClassEntity, but decided not to, because I found them to be very lenghty even withouth it.